AzTS Data Table/Readme.md

```diff
@@ -0,0 +1,273 @@
+|Tags|string|Tags associated with the resource.|{"cognitive": "storage"}|
+
+___
+
+### **Subscription metadata**
+
+|Field Name|Field Type|Description|Sample value|
+|--|--|--|--|
+|subscriptionid|string|The subscription ID.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|Timestamp|datetime|The subscription metadata fetched time.|Fr, 14 Apr 2023 08:39:24 GMT|
+|JobId|int|Job ID for scan (YYYYMMDD).|20230414|
+|SubscriptionName|string|The subscription display name.|Microsoft Azure Test|
+|OrgTenantId|string|The subscription tenant ID.| 00x000xx-00x0-00xx-00xx-0x0xx000xx00|
+|RetryCount|int|Number of retry count to fetch subscription details.|0|
+|Tags|string|Tags attached to the subscription.|{"Env":"Prod"}|
+|State|string|The subscription state.|Enabled|
+|PIMResourceId|string|PIM resource ID of the subscription.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|CreatedDate|datetime|Subscription creatio date and time.|2020-11-11T13:38:39|Subscription creation date.|
+|StateUpdatedOn|datetime|Last date and time on which subscription state is updated. |2021-07-15T12:34:47|
+|IsBillingInfoAvailable|bool|Indicates that subscription billing information is available or not.|true|
+|MaxBillingDate|datetime|Subscription billing date and time.|2023-04-03T00:00:00|
+|SubscriptionOfferType|string|The subcription offer type.|null|,
+|SubscriptionExpiryDate|datime|Subscription expiry date and time.|2025-04-03T00:00:00|
+|CustomField1|string|This field contains custom data.|This field contains custom data.|
+|CustomField2|string|This field contains custom data.|This field contains custom data.|
+|CustomField3|string|This field contains custom data.|This field contains custom data.|
+|CustomField4|string|This field contains custom data.|This field contains custom data.|
+|CustomField5|string|This field contains custom data.|This field contains custom data.|
+|ResourceId|string|The subscription fully qualified ID.|/subscriptions/00000xxx-xx00-0x00-x0x0-00xx00000x00|
+___
+
+### **Assessment data**
+
+|Field Name| Field Type | Description | Sample value |
+|--|--|--|--|
+|JobId|int|Job ID for scan (YYYYMMDD).|20230414|
+| SubscriptionId |string|The subscription ID.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+| AssessmentName | string|The assessment name.| 00000xxx-xx00-0x00-x0x0-00xx00000x00|
+| AssessmentId | string |The assessment ID.| /subscriptions/00000xxx-xx00-0x00-x0x0-00xx00000x00/providers/Microsoft.Security/assessments/00000xxx-xx00-0x00-x0x0-00xx00000x00|
+| AzureResourceId | string |Azure resource Id of the assessed resource.| /subscriptions/00000xxx-xx00-0x00-x0x0-00xx00000x00 |
+| RecommendationDisplayName | string |The recommendation display name.| MFA should be enabled on accounts with read permissions on subscriptions |
+| StatusCode | string |Programmatic code for the status of the assessment.| Healthy |
+| StatusMessage | string |The assessment status message.|{  "code": "Healthy"} |
```

Comment:
We need to format these properly and uniformly

Reply:
If you see the actual tables by going into the branch everything looks good.